README.md

```diff
@@ -17,6 +17,7 @@ Previously, the website and documentation for the Draft U.S. Web Design Standard
   * [Download](#download)
   * [Install using npm](#install-using-npm)
   * [Use another framework or package manager](#use-another-framework-or-package-manager)
+  * [Use the latest version of the `uswds` package](#use-the-latest-version-of-the-uswds-package)
```

Comment:
Ahh, I see — I may have been confused about the nature of the request (I thought both READMEs needed to be updated). But, being that the docs README is updated, I'll go ahead and cancel these changes ...